tests/test_components/test_piping.py

```diff
@@ -83,3 +84,26 @@ def test_Pipe(self):
         # NO TEST NEEDED AT THE MOMENT, THE PIPE PROPERTIES ARE IDENTICAL TO
         # THE PROPERTIES OF THE SIMPLE HEAT EXCHANGER. TESTS ARE LOCATED AT
         # heat_exchanger_tests.py
+
+    def test_Pipeline(self):
+        instance = Pipeline('pipeline')
+
+        self.setup_piping_network(instance)
+
+        # parameter specification
+        self.c1.set_attr(fluid={'H2O': 1}, m=10, p=10, T=220)
+        
+        instance.set_attr(pr=0.99, Tamb = 20, L=1000, D=0.2, 
+                       insulation_m=0.1 ,insulation_tc= 0.035, pipe_thickness=0.002,
+                       material='Steel', 
+                       environment_media= 'air', wind_velocity = 2,
+            ) 
+        self.nw.solve('design')
+        self.nw._convergence_check()
```

Comment:
After this we should make a check for the correctness of the test case, do you have any reference values?